multimodal/setup.py

```diff
@@ -55,6 +55,7 @@
     "openmim>0.1.5,<=0.2.1",
     "defusedxml>=0.7.1,<=0.7.1",
     "albumentations>=1.1.0,<=1.2.0",
+    "jinja2>=3.0.3,<3.2",
```

Comment:
Where do we use `jinja2`?

Reply:
https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/data/templates.py#L18

Reply:
You can refer to the CI error prior to me adding it in this PR: https://github.com/autogluon/autogluon/actions/runs/3914783191/jobs/6692268378#step:4:994

Reply:
This is for supporting prompt source. I’m okay with removing it for now.

Reply:
@sxjscience if you'd like to remove jinja usage, you'll need to open a separate follow-up PR. I believe jinja was a dependency of dask/distributed, which is why we didn't error in the past, but with dask/distributed removed, now we need to explicitly add it as a dependency to AutoMM